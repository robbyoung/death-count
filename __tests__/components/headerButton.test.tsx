import React from 'react';
import renderer from 'react-test-renderer';
import { SolidIcons } from 'react-native-fontawesome';
import HeaderButton from '../../app/components/headerButton';

describe('Header Button Component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<HeaderButton
            onPress={() => undefined}
            icon={SolidIcons.plus}
        />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});