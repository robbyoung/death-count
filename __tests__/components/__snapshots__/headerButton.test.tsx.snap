// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Button Component renders correctly 1`] = `
<View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "color": undefined,
          },
          Object {
            "color": "#ffffff",
            "fontSize": 22,
            "marginRight": 20,
          },
          Object {
            "fontFamily": "FontAwesome5FreeSolid",
          },
        ]
      }
    >
      ÔÅß
    </Text>
  </View>
</View>
`;
